ci/src/main.rs

```diff
@@ -0,0 +1,213 @@
+//! Continuous Integration: rust-bitcoin test runner.
+//!
+//! This program is used by our CI pipeline to test the library. It can, of course, also be used
+//! from the command line also.
+//!
+//! # Hints and tips
+//!
+//! - Build and run (from the repo root directory)
+//!
+//!    `cd ci; cargo build --target-dir target; cd ..; ci/target/debug/ci [OPTIONS]`
+//!
+//! - If you would like more verbose output from cargo consider setting `CARGO_TERM_VERBOSE`.
+//!
+//! - To debug or just see that the tests you think are running are running consider sending
+//!   stdout /dev/null and use `--debug` to see descriptive output.
+//!
+//! - To control the toolchain use `RUSTUP_TOOLCHAIN=nightly`.
+
+use std::process;
+
+use cmd_lib::*;
+use clap::Parser;
+
+/// The list of individual features that we test.
+const FEATURES: &str = "base64 bitcoinconsensus serde rand secp-recovery";
```

Comment:
Why not array?

Reply:
It was either array and join or string and split. So I was lazy on this iteration and just copied and pasted the list from `test.sh`. Can use array if we get serious about this.